maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Models/SetPermissionsRequest.swift

```diff
@@ -4,6 +4,11 @@ enum PermissionValue: String, Codable {
     case allow
     case deny
     case unset
+    case unknown
+        
+    init(from decoder: Decoder) throws {
+        self = try PermissionValue(rawValue: decoder.singleValueContainer().decode(RawValue.self)) ?? .unknown
+    }
```

Comment:
Just to understand the case, was permission handler not handling the non-notification permissions and was failing for other shown permissions?

Reply:
It was failing when the value was `never` when decoding. and since we only have `allow` `deny` `unset` supported for permissions it makes sense not to fail if some other value is recieved for other permissions